src/Splunk.OpenTelemetry.AutoInstrumentation.Plugin/Configuration/CompositeConfigurationSource.cs

```diff
@@ -0,0 +1,117 @@
+﻿// <copyright file="CompositeConfigurationSource.cs" company="Splunk Inc.">
```

Comment:
It is from OTel. We need to keep the original license header. See how it is done e.g. in https://github.com/signalfx/splunk-otel-dotnet/blob/main/test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/SmokeTests.cs

Same comment applies to other places